cliquet/storage/postgresql/__init__.py

```diff
@@ -448,8 +448,48 @@ def delete_all(self, collection_id, parent_id, filters=None,
             record[deleted_field] = True
             records.append(record)
 
+        if delete_tombstones:
+            self.delete_tombstones(collection_id, parent_id,
+                                   id_field=id_field,
+                                   modified_field=modified_field,
+                                   auth=auth)
+
         return records
 
+    def delete_tombstones(self, collection_id, parent_id, before=None,
+                          id_field=DEFAULT_ID_FIELD,
+                          modified_field=DEFAULT_MODIFIED_FIELD,
+                          auth=None):
+        query = """
+        WITH deleted_records AS (
+            DELETE
+            FROM deleted
+            WHERE parent_id = %%(parent_id)s
+              AND collection_id = %%(collection_id)s
+              %(conditions_filter)s
+            RETURNING id
+        )
+        SELECT COUNT(*) as deleted FROM deleted_records
+        RETURNING deleted;
+        """
```

Comment:
I think you can just DELETE FROM and use cursor.rowcount to get the number of affected rows